server/api/validators/geojson.js

```diff
@@ -50,6 +94,20 @@ exports.point = function() {
   };
 }
 
+function validateBboxStringCoordinate(ctx, coordinates, i, callback) {
+  return ctx.validate(coordinateCtx => {
+
+    // Make the error indicate the index of the invalid coordinate
+    // within the bbox string, e.g. "bbox[1]".
+    coordinateCtx.set({
+      location: `${coordinateCtx.get('location')}[${i}]`,
+      value: coordinates[i]
+    });
+
+    return callback(coordinateCtx, i);
```

Comment:
Why is `i` passed to the callback ?
As defined at line 42-45 of the same file, the given callbacks only requests one argument, which is `coordinateCtx`.

Reply:
I thought I would need it when I wrote it. Will remove.